s3-shield/cloud-init/generate_yaml.sh

```diff
@@ -3,11 +3,13 @@
 # cd where this script it
 cd $(dirname "$0")
 
-BASE64_VCL=$(base64 -w 0 "../default.vcl")
-BASE64_S3CONF=$(base64 -w 0 "s3.conf")
+BASE64_VCL=$(gbase64 -w 0 "../default.vcl")
```

Comment:
why the `base64`->`gbase64` change?
from my snooping around, `gbase64` is the `brew`-installed alias for `base64`, if I'm correct, I'd prefer sticking to the original name

Reply:
@gquintard my bad! The `base64` implementation on my Mac behaves different than the Linux one. That's why I locally use `gbase64`. However, I forgot to undo this change prior to committing.

Should be fixed now.